mosu-app/src/widgets/auth/SignupVerificationStep.tsx

```diff
@@ -0,0 +1,78 @@
+import { useCallback, useEffect, useState } from "react";
+import { useFormContext } from "react-hook-form";
+import { toast } from "react-toastify";
+
+import { SignUpFormData } from "@/features/auth/models/SignupSchema";
+import { getTrCert } from "@/features/auth/services/getTrCert";
+import { SignUpStep } from "@/features/auth/types";
+
+import { Spinner } from "@/shared/components/Spinner/Spinner";
+import { useSubmitForm } from "@/shared/hooks/useSubmitForm";
+
+export const SignUpVerificationStep = (
+    {
+        // useStep,
+    }: {
+        useStep: {
+            step: SignUpStep;
+            setStep: (nextStep: SignUpStep) => void;
+        };
+    },
+) => {
+    const { getValues } = useFormContext<SignUpFormData>();
+
+    const [isPending, setIsPending] = useState<boolean>(true);
+    const [trCert, setTrCert] = useState<string>("");
+
+    const [loginId, password, ...termsOfUse] = [
+        getValues("id"),
+        getValues("password"),
+        getValues("termsOfUse"),
+        getValues("termsOfInfo"),
+        getValues("termsOfMarketing"),
+    ];
```

Comment:
```suggestion
    const { 
    id: loginId, 
    password, 
    termsOfUse, 
    termsOfInfo, 
    termsOfMarketing 
} = getValues();
```
이렇게도 쓸 수 있는 거로 알아욤
아니면
```suggestion
const formValues = getValues();
const { id: loginId, password, termsOfUse, termsOfInfo, termsOfMarketing } = formValues;
```